monkey/monkey_island/cc/ui/src/components/attack/techniques/T1107.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import '../../../styles/Collapse.scss'
+import ReactTable from "react-table";
+import { renderMachineFromSystemData } from "./Helpers"
+
+
+class T1107 extends React.Component {
+
+  constructor(props) {
+    super(props);
+  }
+
+  static renderDelete(status){
+    if(status === 2){
+      return <span>Yes</span>
+    } else {
+      return <span>No</span>
+    }
+  }
+
+  static getDeletedFileColumns() {
+    return ([{
+      columns: [
+        {Header: 'Machine', id: 'machine', accessor: x => renderMachineFromSystemData(x._id.machine), style: { 'whiteSpace': 'unset' }},
+        {Header: 'Path', id: 'path', accessor: x => x._id.path, style: { 'whiteSpace': 'unset' }},
+        {Header: 'Deleted?', id: 'deleted', accessor: x => this.renderDelete(x._id.status),
```

Comment:
Why would we want to show files which weren't deleted? Just to show we failed doing so?

Reply:
I think it's unlikely, but possible to allow read and write but not delete to certain users/folders. In that case we'd want to show that we tried to delete, but failed.

Reply:
fair enough